### PR TITLE
Update requirements for 0.9.5

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by ltoenning** [comment]
First, thanks for your contribution!

Could you summarize which changes this pull-request introduces? If I see correctly all these commits are from @oktal3700 and @ltoenning and are already within the homepage branch. Correct me, if I overlooked something.

Furthermore, this pull request is marked to be merged into the master branch. For this repository, we use Github Pages for hosting. Everything within this repository on the master branch is directly displayed here: https://swift-project.org. 
Because we use the Hugo framework the "sources" need to be built for the resulting final homepage. Hence only our Github Actions pipeline (which builds/deploys the website) should commit to master. Every other content-related work should be done on the homepage branch.

**Comment by oktal3700** [comment]
Hi @CPTShawnx 

It's not clear what you're trying to achieve. If you want to help improve the swift project website you are very welcome to. I suggest you join us on our [Discord server](https://discord.gg/R7Atd9A) to discuss it. For now, I'm closing this pull request as it contains no changes and is indistinguishable from spam.
